src/styles/components/_profile-card.scss

```diff
@@ -0,0 +1,109 @@
+@import '../abstracts/_abstracts-dir';
+
+.profile-card {
+  display: flex;
+  justify-content: flex-end;
+
+  @include respond-above('md') {
+    width: 30%;
+  }
+
+  &--background-image {
+    background: {
+      image: url('/images/blue-ellipse.png');
```

Comment:
Please rename the file to something like `profile-card-background` or any other meaningful names.